packages/react-native-sdk/src/components/LobbyView.tsx

```diff
@@ -18,11 +18,12 @@ import { LOCAL_VIDEO_VIEW_STYLE } from '../constants';
 
 const ParticipantStatus = () => {
   const connectedUser = useConnectedUser();
+  const participantLabel = connectedUser?.name ?? connectedUser?.id;
```

Comment:
This the reason why I say lets pass id as username for custom user..

we cannot ask every other DF apps to do this

Reply:
Done